sormas-ui/src/main/java/de/symeda/sormas/ui/customexport/ExportConfigurationsGrid.java

```diff
@@ -68,37 +86,74 @@ private void buildGrid(List<DataHelper.Pair<String, ExportGroupType>> availableP
 			.setCaption(I18nProperties.getPrefixCaption(ExportConfigurationDto.I18N_PREFIX, ExportConfigurationDto.NAME))
 			.setExpandRatio(1);
 
-		addComponentColumn((config) -> this.buildButtonLayout(config, availableProperties, propertyCaptionProvider)).setId(COLUMN_ACTIONS)
+		addComponentColumn((config) -> this.buildButtonLayout(config, availableProperties, propertyCaptionProvider, true))
+			.setId(COLUMN_ACTIONS)
+			.setCaption("");
+	}
+
+	private void buildGridPublicExportCustomConfiguration(
+		List<DataHelper.Pair<String, ExportGroupType>> availableProperties,
+		Function<String, String> propertyCaptionProvider) {
+
+		setSelectionMode(SelectionMode.NONE);
+		setHeightMode(HeightMode.ROW);
+
+		addColumn(ExportConfigurationDto::getName)
+			.setCaption(I18nProperties.getPrefixCaption(ExportConfigurationDto.I18N_PREFIX, Captions.ExportConfiguration_sharedToPublic))
+			.setExpandRatio(1);
+
+		addComponentColumn((config) -> this.buildButtonLayout(config, availableProperties, propertyCaptionProvider, false))
+			.setId(COLUMN_ACTIONS_PUBLIC)
 			.setCaption("");
 	}
 
 	public void reload() {
 		List<ExportConfigurationDto> configs =
-			FacadeProvider.getExportFacade().getExportConfigurations(new ExportConfigurationCriteria().exportType(exportType));
+			FacadeProvider.getExportFacade().getExportConfigurations(new ExportConfigurationCriteria().exportType(exportType), false);
 		setItems(configs);
 		setHeightByRows(configs.size() > 0 ? (Math.min(configs.size(), 10)) : 1);
 	}
 
+	public void reloadPublicExportCustomConfiguration() {
```

Comment:
having a flag on the class, like isForPublic this duplicate method could be avoided